benchmarks/Cargo.toml

```diff
@@ -55,7 +55,7 @@ pprof = { version = "0.13", features = [
 
 [features]
 default = ["parallel", "mimalloc", "bench-metrics"]
-bench-metrics = ["openvm-native-recursion/bench-metrics"]
+bench-metrics = ["openvm-native-recursion/bench-metrics", "openvm-native-compiler/bench-metrics"]
```

Comment:
we should make `bench-metrics` feature in `native-recursion` also turn on `native-compiler/bench-metrics`

Reply:
@stephenh-axiom-xyz ok to merge after this